htmlvalidator/middleware.py

```diff
@@ -1,5 +1,8 @@
 from django.conf import settings
-from django.contrib.sites.models import RequestSite
+try:
+    from django.contrib.sites.requests import RequestSite
+except Importerror:  # Django < 1.7
```

Comment:
No, the comment is correct, this ImportError will only occur with < 1.7. django.contrib.sites.requests.RequestSite was introduced in 1.7.

https://docs.djangoproject.com/en/1.6/ref/contrib/sites/#requestsite-objects
https://docs.djangoproject.com/en/1.7/ref/contrib/sites/#requestsite-objects